consensus/spos/bls/subroundSignature.go

```diff
@@ -192,11 +192,11 @@ func (sr *subroundSignature) completeSignatureSubRound(
 		return false
 	}
 
-	if shouldWaitForAllSigsAsync {
-		if sr.EnableEpochHandler().IsFlagEnabled(common.ConsensusModelV2Flag) {
-			sr.AddProcessedHeadersHashes(processedHeaderHash, index)
-		}
+	if sr.EnableEpochHandler().IsFlagEnabled(common.ConsensusModelV2Flag) {
+		sr.AddProcessedHeadersHashes(processedHeaderHash, index)
+	}
 
+	if shouldWaitForAllSigsAsync {
```

Comment:
Before this change, `sr.AddProcessedHeadersHashes(processedHeaderHash, index)` was only executed if both conditions:
1. `sr.EnableEpochHandler().IsFlagEnabled(common.ConsensusModelV2Flag)`
2. `shouldWaitForAllSigsAsync`
were met.

Not it only requires the 1. one. Is that intended?

Reply:
in the previous version, with the multikey, the` sr.AddProcessedHeadersHashes(processedHeaderHash, index) `will be called just on the leader (`shouldWaitForAllSigsAsync=true`) and no other signatures are going to be recorded. 
Because the bitmap is created from the ProcessedHeaderHashes it is necessary to enter `AddProcessedHeadersHashes` for every managed BLS key on the multikey setup